kratos/processes/levelset_convection_process.h

```diff
@@ -225,19 +248,76 @@ class LevelSetConvectionProcess
             const double Nnew_before = 1.0 - Nold_before;
 
             // Emulate clone time step by copying the new distance onto the old one
-            #pragma omp parallel for
-            for (int i_node = 0; i_node < static_cast<int>(mpDistanceModelPart->NumberOfNodes()); ++i_node){
-                auto it_node = mpDistanceModelPart->NodesBegin() + i_node;
+            if (mBfeccOrder == 0){
+                #pragma omp parallel for
```

Comment:
Parallel utils.

Reply:
I need direct access to "i_node". It is possible to use "for_each" but it would not be clean.

Reply:
You can use the `block_for_each`  with the `.Nodes()` container.

Reply:
Should I create "i_node" with (Node.Id - 1)?

Reply:
mmm... if you do this we rely that the nodal Ids. are consecutive, which might not be the case. Using `std::distance` might do the job (however I'm afraid that we will need to iterate one of the `mVelocity`-type lists, as the `Nodes` container doesn't have this method...) .